tests/unit/batch_sys_handlers/test_slurm.py

```diff
@@ -33,8 +36,14 @@
             },
             [
                 '#SBATCH --job-name=axe.1.chop',
-                '#SBATCH --output=cylc-run/chop/log/job/1/axe/01/job.out',
-                '#SBATCH --error=cylc-run/chop/log/job/1/axe/01/job.err',
+                (
+                    f'#SBATCH --output='
+                    f'{home}cylc-run/chop/log/job/1/axe/01/job.out'
```

Comment:
```suggestion
                    f'{home}/cylc-run/chop/log/job/1/axe/01/job.out'
```
This is a crushingly trivial point, but sightly nicer to read (and a tiny bit safer) to put a path separator here, and (optionally) remove it from `os.path.expandvars("$HOME")` above)